src/Core/Checkout/Payment/PaymentProcessor.php

```diff
@@ -81,7 +81,7 @@ public function pay(
 
             return $response;
         } catch (\Throwable $e) {
-            $this->logger->error('An error occurred during processing the payment', ['orderTransactionId' => $transaction->getId(), 'exceptionMessage' => $e->getMessage()]);
+            $this->logger->error('An error occurred during processing the payment', ['orderTransactionId' => $transaction->getId(), 'exceptionMessage' => $e->getMessage(), 'exceptionTrace' => $e->getTraceAsString()]);
```

Comment:
the PSR kinda defines an `'exception'` key for the context. should we just fill that, instead of separate message and trace? :thinking:

Reply:
Hmm, maybe in addition? In our default log files, this doesn't help traceability.

Reply:
ah right, in the files it is not really enrolled :+1: